tests/sentry/api/endpoints/test_organization_repository_details.py

```diff
@@ -286,51 +286,6 @@ def test_put_hide_repo_with_commits(self):
         assert repo.status == ObjectStatus.HIDDEN
         assert len(Commit.objects.filter(repository_id=repo.id)) == 0
 
-    def test_put_cancel_deletion_duplicate_exists(self):
```

Comment:
removed this test because with the remaining constraint, two repos with the same `external_id`, `organization_id`, and `provider` (not specified in this test) shouldn't exist